src/components/Button.js

```diff
@@ -0,0 +1,30 @@
+import {StyleSheet, Text, TouchableOpacity} from 'react-native';
+import React from 'react';
+
+const Button = ({text, color, onHandlePress}) => {
+  return (
+    <TouchableOpacity
+      onPress={onHandlePress}
+      style={[styles, {backgroundColor: color}]}>
+      <Text style={textStyles}>{text}</Text>
+    </TouchableOpacity>
+  );
+};
+
+export default Button;
+
+const styles = StyleSheet.create({
+  marginTop: 10,
+  alignItems: 'center',
+  justifyContent: 'center',
+  borderRadius: 24,
+  height: 72,
+  width: 332,
+  alignSelf: 'center',
+});
+
+const textStyles = StyleSheet.create({
+  color: 'white',
+  fontWeight: '700',
+  fontSize: 23,
+});
```

Comment:
You do not have to create separate objects for each style and this can be tiring as well, you can achieve same functionality with only one styles object.

i.e.

```
const styles = StyleSheet.create({
         button: {
                      marginTop: 10,
                      alignItems: 'center',
                      .....
            },
         text: {
               color: 'white',
               ....
          }     
})
```

And you can use these as 

`style={[styles.button, styles.text]}`

Always try to avoid passing inline styles as much as you can.

Reply:
in inline styling part of the comment, we don't have to color prop inside the styles object. that's why it's inline. is there any way of making it available outside?

Reply:
mark it as resolved